src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	/**
+	 * Test Utils isAnyNull method
+	 */
+	
+	@Test
+	public void isAnyNull_returnsTrue() {
```

Comment:
the test name should have 3 parts `<method being tested>_<what is being tested>_<what is the expected return value>`
you can eliminate part 3 and 2 only if you have broadened the scope of the test to include testing multiple things.